activities/Chess.activity/js/chessGame.js

```diff
@@ -391,18 +391,24 @@ var ChessGame = {
       var start_time = Date.now();
       var possibleMove = this.state.findmove(this.level);
       var delta = Date.now() - start_time;
+
+      // console.log(`Random Move: ${possibleMove}, Delta: ${delta}`);
+
       // game over
-      if (possibleMove.length === 0) {
+      if (possibleMove.length === 0 && (move.flags === 7 || move.flags === 15 || move.flags === 23 || move.flags === 39)) {
+        // console.log("Game won by computer");
         this.game_won = true;
         return;
       }
-      if (1 / possibleMove[2] == 1 / this.state.stalemate_scores[this.state.to_play]) {
+  
+      if (possibleMove.length === 0 && (move.flags === 5 || move.flags === 13 || move.flags === 21 || move.flags === 37 || move.flags === 64)) {
```

Comment:
Not sure to understand this test. Could you explain?

Reply:
I removed this line of code, it was not needed is what I realised later